packages/sdk/src/create/libp2p.ts

```diff
@@ -138,12 +138,15 @@ function configureNetworkOptions(
     options.shardInfo = { contentTopics: options.contentTopics };
   }
 
+  if (!options.shardInfo) {
+    options.shardInfo = DefaultShardInfo;
+  }
+
   const shardInfo = options.shardInfo
     ? ensureShardingConfigured(options.shardInfo)
     : undefined;
 
-  options.pubsubTopics = shardInfo?.pubsubTopics ??
-    options.pubsubTopics ?? [DefaultPubsubTopic];
+  options.pubsubTopics = options.pubsubTopics ?? shardInfo?.pubsubTopics;
```

Comment:
This is a good point but I don't think we should get rid of `pubsubTopics` for internal use because we:
- have tight coupling to it in our code base;
- libp2p itself uses it a lot;
- it's native to gossipSub so anyway we face the need to refer to it one way or another;